prime-router/docs/getting_started.md

```diff
@@ -236,6 +236,7 @@ End-to-end tests check if the deployed system is configured correctly.  The test
     ./gradlew primeCLI --args='create-credential --type=UserPass --persist=IGNORE--CSV --user foo --pass pass'
     ./gradlew primeCLI --args='create-credential --type=UserPass --persist=IGNORE--HL7 --user foo --pass pass'
     ./gradlew primeCLI --args='create-credential --type=UserPass --persist=IGNORE--HL7-BATCH --user foo --pass pass'
+    ./gradlew primeCLI --args='create-credential --type=UserPass --persist=DEFAULT-FTP --user foo --pass pass'
```

Comment:
See note above about naming this `DEFAULT-SFTP`. 

Also, I'm inclined to wrap this all into a gradle task that will refresh all the permissions in our local dev environment. Thoughts @jimduff-usds @carlosfelix2?

Reply:
That would be nice to have for bootstrapping the dev environment, so we don't have to think about it during local setup.

Reply:
Definitely, although I think this could be a good first step, and do that as a followup?

Reply:
Is the intention to create a gradle task that does the same thing than ?
"primeCLI --args='create-credential --type=UserPass --persist=DEFAULT-FTP --user foo --pass pass'

Or a task that iterates over the settings table to update each receiver that has SFTP as the transport?

Reply:
@pablocaviglia - the first one.   Just wrap
   `primeCLI --args='create-credential --type=UserPass --persist=DEFAULT-FTP --user foo --pass pass`
with a gradle call.  

Important:   set it up so you can pass in the user and password as parameters.